hack/pseudo-version/pseudo-version.go

```diff
@@ -81,14 +81,14 @@ func main() {
 
 	switch {
 	case *printSemVer:
-		fmt.Println(*base)
+		fmt.Print(*base)
 	case *printTimestamp:
-		fmt.Println(headTime.Format("20060102150405"))
+		fmt.Print(headTime.Format("20060102150405"))
 	case *printBranch:
-		fmt.Println(parsedBranch)
+		fmt.Print(parsedBranch)
 	case *printReleaseVersion:
-		fmt.Println(strings.TrimPrefix(rawBranch, "release/"))
+		fmt.Print(strings.TrimPrefix(rawBranch, "release/"))
 	default:
-		fmt.Println(version)
```

Comment:
What's the reason for this change? I don't think it is a good idea to do this in general.

Reply:
The output is used directly written into the variable and used e.g. [here](https://github.com/edgelesssys/constellation/pull/1281/files#diff-1e7de1ae2d059d21e1dd75d5812d5a34b0222cef273b7c3a2af62eb747f9d20aR22). With the trailing newline that would break. It feel good to change that `Println` but wasn't quite sure how to handle it differently.